test/regression/targets/sc-xxxxx.cc

```diff
@@ -0,0 +1,1178 @@
+#include <chrono>
+#include <climits>
+#include <thread>
+
+#include <tiledb/tiledb>
+#include <tiledb/tiledb_experimental>
+
+#include <test/support/tdb_catch.h>
```

Comment:
```suggestion
#include <catch2/catch_template_test_macros.hpp>
```

This will drastically reduce the number of Catch2 headers included. It might be a good idea to use specific Catch2 headers in new code and sunset `tdb_catch.h`, I have a branch that replaces it across the codebase.